documentation/SupportingDocuments/Checklist_for_API_description_review.md

```diff
@@ -0,0 +1,16 @@
+# Checklist for API description reviews
+
+
+| **Section** | **Checkpoints** |
+|---|---|
+| API Description | <ul><li>API targets and functionality is described complete and understandable</li><li>Visual illustrating API functionality and integration is included</li></ul> |
+| Use Cases | <ul><li>Use cases are described understandable</li><li>Use case name in bold, rest in normal font</li></ul> |
+| Benefits | <ul><li>Benefits are described understandable</li><li>Benefit name in bold, rest in normal font</li></ul> |
+| API Portfolio: | <ul><li>Prefix correct (see column section)</li><li>Correct portfolio is shown</li></ul> |
+| SubProject Wiki: | <ul><li>Prefix correct (see column section)</li><li>Link is correct and named with SubProject or Independent Sandbox</li><li>"(incl. how to meet the team)" is added</li></ul> |
```

Comment:
There are many ways to write "Sub Project" within CAMARA, but the official one from the ProjectStructureAndRoles is "Sub Project". A CamelCase word does not exist in normal language.

```suggestion
| Sub Project Wiki: | <ul><li>Prefix correct (see column section)</li><li>Link is correct and named with Sub Project or Independent Sandbox</li><li>"(incl. how to meet the team)" is added</li></ul> |
```

BTW: in case of an Independent Sandbox which is not part of a Group (like in case of SessionInsights which is already grouped under Connectivity Insights), Sub Project Wiki and API Wiki are the same ... is this intended?